ts/svelte/svelte.ts

```diff
@@ -243,7 +239,7 @@ async function extractSvelteAndDeps(
 /// may be coming from the source folder, which breaks ./foo imports.
 /// Adjust the path to make it appear they're all in the same folder.
 function remapBinToSrcDir(file: string): string {
-    return file.replace(new RegExp("bazel-out/[-_a-z]+/bin/"), "");
+    return file.replace(new RegExp(".bazel/out/[-_a-z]+/bin/"), "");
```

Comment:
I think this change may not be correct. svelte.ts executes inside Bazel's build structure, which contains a bazel-out folder. The recent rename was just of the convenience symlinks, which are only used for accessing Bazel's files outside of Bazel. Code that runs as part of the Bazel build process uses the internal tree, which still uses the old names.

```
Work/code/dtop% ls -l .bazel/dtop/bazel-out
drwxr-xr-x  - dae 23 Jan 15:27 _actions
drwxr-xr-x  - dae 26 Jan 21:16 _tmp
[...]
```